arcade/examples/sprite_health.py

```diff
@@ -1,230 +1,249 @@
 """
-Sprite Hit Points and Health Bars
+Sprite Health Bars
 
 Artwork from https://kenney.nl
 
 If Python and Arcade are installed, this example can be run from the command line with:
 python -m arcade.examples.sprite_health
 """
-import random
+import math
 import arcade
 
 SPRITE_SCALING_PLAYER = 0.5
-SPRITE_SCALING_COIN = 0.2
-SPRITE_SCALING_LASER = 0.8
-COIN_COUNT = 50
+SPRITE_SCALING_ENEMY = 0.5
+SPRITE_SCALING_BULLET = 1
+INDICATOR_BAR_OFFSET = 32
+ENEMY_ATTACK_COOLDOWN = 0.5
```

Comment:
Playing this with the current values is a bit frantic and overwhelming, especially with a touchpad.

I think you should do one of the following to make the example easier to understand on first launch:
1. Increase `ENEMY_ATTACK_COOLDOWN` to 1.0, and decrease `PLAYER_HEALTH` to 5
2. Set `BULLET_SPEED` to 2

I tried playing both changes locally, and I think either is ok. You could also combine the approaches, but you will need to fiddle with the values to find something that plays reasonably fast without being overwhelming.